README.rst

```diff
@@ -7,40 +7,38 @@ Make open data sharing a no-brainer for ethics committees.
 Statement of the problem
 ========================
 
-The ideology of open and reproducible science makes its ways into
-various fields of science.  Neuroimaging is a driving force today
-behind many fields of brain sciences.  Despite possibly terabytes of
-neuroimaging data collected for research daily, just a small fraction
-becomes publicly available. Partially it is because management of
-neuroimaging data requires to confirm to established legal norms,
+The ideology of open and reproducible science makes its ways into various fields
+of science.  Neuroimaging is a driving force today behind many fields of brain
+sciences. Despite possibly terabytes of neuroimaging data collected for research
+daily, just a small fraction becomes publicly available. Partially it is because
+management of neuroimaging data requires to confirm to established legal norms,
```

Comment:
note that due to paragraph reformatting it is hard to impossible to see any actual differences

Reply:
I will push this change to also demo our new feature -- RTD test builds in PRs ;)

Reply:
the translations were made mostly by non github people - often MD checking with their data officer (tiny variations here and there to reflect local rules) ;; I'd quite like to move forward as see changes on readthedoc so we can submit the paper (but understand you are cautious)

Reply:
ok, then -- let's merge but I would still like to get at least #43 (or better with #42) before we "roll it out"